src/extensions/averagewaveforms/AverageWaveformsNew.tsx

```diff
@@ -7,6 +7,7 @@ import AverageWaveformPlotNew from './AverageWaveformPlotNew';
 const averageWaveformsCalculationPool = createCalculationPool({maxSimultaneous: 6});
 
 const AverageWaveformsNew: React.FunctionComponent<SortingViewProps> = (props) => {
+    const { selection } = props
```

Comment:
These are repeat changes due to the code sync. I'm sort of wondering if we should avoid checking in both sets of files into the repo. I guess the sync could be part of the build step. I'm not sure. We'll leave it as is for now.

Reply:
Yeah, maybe some kind of build hook would be a good idea? As long as unison is able to harmonize the files successfully. (I would hope that we won't have too much temptation to edit both copies of the file, but I guess you never know.)